plugin/cmd.go

```diff
@@ -62,7 +63,7 @@ func cmdFunc(flags caddycmd.Flags) (int, error) {
 	options := createOptions(flags)
 
 	if options.Mode&config.Server == config.Server {
-		log.Printf("[INFO] Running caddy proxy server")
+		logger().Info("Running caddy proxy server")
```

Comment:
Should we cache this logger in global constant instead of recreating it every time we need it?

Reply:
Maybe not globally, but at least locally
Because caddy will change the logger every time a caddyfile is reloaded.

Reply:
yup, one mo :)